com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -349,6 +388,23 @@ public IReadOnlyList<ulong> ConnectedClientsIds
 
         internal void InvokeOnClientConnectedCallback(ulong clientId) => OnClientConnectedCallback?.Invoke(clientId);
 
+        public event Action OnReadyCallback = null;
```

Comment:
It might be a good idea to add XML Doc summary for public facing API documentation reference purposes.
_(apply this to all public facing additions)_